go/chat/bots/commands.go

```diff
@@ -152,8 +153,14 @@ func (b *CachingBotCommandManager) createConv(ctx context.Context, param chat1.A
 			return res, errors.New("missing team name")
 		}
 		topicName := fmt.Sprintf("___keybase_botcommands_team_%s_%v", username, param.Typ)
+		var membersType chat1.ConversationMembersType
+		if _, err := teams.Details(ctx, b.G().GlobalContext, *param.TeamName); err == nil {
```

Comment:
There is likely a faster way to do this, this particular call does a ton of stuff. Just use normal team loader